bgpd/bgp_flowspec.c

```diff
@@ -0,0 +1,83 @@
+/* BGP FlowSpec for packet handling
```

Comment:
See this is exactly what I was talking about @vjardin6WIND @pguibert6WIND You are claiming copyright to the work that this person in China did.  I realize that this person didn't include a copyright notice but this person clearly wrote this work.  You need to give credit here.

Reply:
I don't know what to do here.  This file is fundamentally different than what the person in China has. so I dunno.

Reply:
This code is fundamentally different. <sigh>.  If you are happy with the cpyright notification then I am as well.  Since you are the one clamiing it.  I'll back off

Reply:
I will keep it because the main decoding mechanism ( to decode nlri) is kept.
even if the code moved from place, that algo is kept

Reply:
I believe the correct thing to do here is to use `Portions:` and list all the contributing parties, as described under ["Adding copyright claims to existing files"](http://45.33.5.119/dev-guide/workflow.html#adding-copyright-claims-to-existing-files) in both COMMUNITY.md and the developer docs. Of course this is assuming the code is under an FRR-compatible license but I'm sure you already checked that.

Reply:
done, @qlyoung , by using portions